packages/shared/src/hooks/post/useViewPost.ts

```diff
@@ -3,39 +3,31 @@ import {
   useMutation,
   useQueryClient,
 } from '@tanstack/react-query';
-import { Post, sendViewPost } from '../../graphql/posts';
+import { getDay } from 'date-fns';
+import { sendViewPost } from '../../graphql/posts';
 import { generateQueryKey, RequestKey } from '../../lib/query';
 import { useAuthContext } from '../../contexts/AuthContext';
 import { ReadingDay, UserStreak } from '../../graphql/users';
 
-export const useViewPost = (
-  post: Post,
-): UseMutateAsyncFunction<unknown, unknown, string> => {
+export const useViewPost = (): UseMutateAsyncFunction<
+  unknown,
+  unknown,
+  string
+> => {
   const client = useQueryClient();
   const { user } = useAuthContext();
   const streakKey = generateQueryKey(RequestKey.UserStreak, user);
   const readKey = generateQueryKey(RequestKey.ReadingStreak30Days, user);
   const { mutateAsync: onSendViewPost } = useMutation({
     mutationFn: sendViewPost,
     onSuccess: async () => {
-      await client.invalidateQueries({ queryKey: streakKey });
```

Comment:
Before was kinda strange:
- Invalidate query
- Override values with local one

But if we invalidate query we want to refetch, not override with local "non-safe" values. 
`post.read` is not consistent through the queries and we cannot rely on this property for a streak update.